bigtable-hbase-dataflow/src/main/java/com/google/cloud/bigtable/dataflow/CloudBigtableConnectionPool.java

```diff
@@ -60,6 +60,13 @@ protected synchronized Connection getConnection(Configuration config, String key
 
   @VisibleForTesting
   protected Connection createConnection(Configuration config) throws IOException {
```

Comment:
I would love to see at least a comment on `getConnection` telling the user (developer) that he does not need to close the connection. Otherwise (out of usual practise) the user might be tempted/assume that he has do close the connection at the end of its DoFn or during `finsihBundle`